### PR TITLE
Docs/release process

### PR DESCRIPTION
**Pull request by EstebanDalelR**
## Description
Added release process clarity
## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)  
- [ ] New feature (non-breaking change which adds functionality)  
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)  
- [x] Documentation update  
- [ ] Chore: cleanup/renaming, etc  
- [ ] RFC  
## Acceptance

- [x] I have read [the Contributing guidelines](CONTRIBUTING.md)
- [x] I have read the [Code of Conduct](CODE_OF_CONDUCT.md)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Contributing to Watermelon's extension development
 
-For now, we discuss on [Slack](https://join.slack.com/t/watermelonusers/shared_invite/zt-18vhxghk6-7YSg30oWqQSZqdOzlkR~Rw). 
+For now, we discuss on [Slack](https://join.slack.com/t/watermelonusers/shared_invite/zt-18vhxghk6-7YSg30oWqQSZqdOzlkR~Rw).
```
Comment (baristaGeek) [inline CONTRIBUTING.md:3]: Discord [https://discord.gg/H4AE6b9442](https://t.co/fMIlnb9egq)

**Review by baristaGeek — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (EstebanDalelR) [reply CONTRIBUTING.md:3]: Interesting, why wasn't this changed.
Seems most contributors are on Slack now.

Comment (baristaGeek) [reply CONTRIBUTING.md:3]: We're now putting all our efforts on Discord

Comment (EstebanDalelR) [reply CONTRIBUTING.md:3]: Why wasn't this changed before?

Comment (baristaGeek) [reply CONTRIBUTING.md:3]: Because I missed it.

**Review by baristaGeek — APPROVED** [review]

**Pull request merged**